### DISCUSSION TITLE
ERROR: FAIL could not package project - v = InvocationError(".../.tox/.package/bin/python -m pip install spacy==2.3.4 ...

### DISCUSSION THREAD

**Discussion by james811223ad** [explosion/spaCy#11604] [Help: Installation] [answered]
I'm trying to install Spacy 2.3.4, but getting an error as shown below. Does anyone know what should I do? I'm using Python 3.10.
```
ERROR: FAIL could not package project - v = InvocationError("/home/runner/work/.../.tox/.package/bin/python -m pip install spacy==2.3.4 'joblib<2' 'boto3<2' 's3fs<1' 'PyPDF2<3' 'datefinder<1' 'scikit-learn<2' 'us<3' 'rapidfuzz<3' 'pandas<2' 'uszipcode<1' 'aioredis<2' 'responses<1' 'requests<3' 'psutil<6' 'Pillow<10' 'nltk<4'", 1)
Error: Process completed with exit code 2.
```

**Comment by adrianeboyd** [comment, marked as answer]
That's not enough of an error message for us to help with the specific problem. Maybe there's a verbose option that would show more? (Actually, don't spend too much time on this. My first guess is that it's related to generated cython code in older releases that's incompatible with newer versions of python. It's not 100% impossible to install this version of spacy with python 3.10, but it's going to involve installing from github/source instead of `pip install spacy==2.3.4`.)

In general you will have a much easier time if you use a version of python that was supported and tested at the time of release. For this version, it looks like python 3.9 was the most recent version. You can look in the left-hand menu in the pypi release page for more details about the specific versions of python that were supported: https://pypi.org/project/spacy/2.3.4/

**Reply by adrianeboyd** [reply]
Also it would usually make sense to use the most recent patch release, which is currently v2.3.7 instead of v2.3.4.
